src/hooks/useUserByAddress.ts

```diff
@@ -0,0 +1,19 @@
+import { useGetUserByAddressQuery } from '~gql';
+
+const useUserByAddress = (address: string) => {
+  const { data, error, loading } = useGetUserByAddressQuery({
+    variables: {
+      address,
+    },
+    fetchPolicy: 'cache-and-network',
```

Comment:
Don't think we need to specify this here, either? The default of cache-first should be fine for the user.